.github/ISSUE_TEMPLATE/rule_proposal.yml

```
@@ -0,0 +1,38 @@
+name: Rule proposal
+description: Have an idea for a new rule?
+title: "Rule proposal: "
+labels:
+  - new rule
+  - evaluating
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening a new issue:
+        - Look for existing [open or closed rule proposals](https://github.com/sindresorhus/eslint-plugin-unicorn/issues?q=is%3Aissue+label%3A%22new+rule%22)
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Description
+      description: Explain here why this rule would be beneficial
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Fail
+      description: Specify an example of code that should be detected
```

```suggestion
      description: Specify examples of code that should be detected
```

Normally, the examples should be more than one.

---

I think it's even better if we add another code block in `value`